lib/rspec/rails/extensions/active_support/json.rb

```diff
@@ -0,0 +1,63 @@
+if defined?(::RSpec::Mocks)
```

Comment:
rspec-core waits to load rspec-mocks until just before the first example group is defined:

https://github.com/rspec/rspec-core/blob/v3.2.0/lib/rspec/core/example_group.rb#L376-L384
https://github.com/rspec/rspec-core/blob/v3.2.0/lib/rspec/core/example_group.rb#L435-L443
https://github.com/rspec/rspec-core/blob/v3.2.0/lib/rspec/core/configuration.rb#L1209-L1212
https://github.com/rspec/rspec-core/blob/v3.2.0/lib/rspec/core/configuration.rb#L412
https://github.com/rspec/rspec-core/blob/v3.2.0/lib/rspec/core/configuration.rb#L535

Thus, rspec-mocks may not be loaded at the point this file is loaded.  I think this could go in a `before(:suite)` hook in order to be delayed until after the point where rspec-mocks will have been loaded if it is going to be.